doc/apidoc.rst

```diff
@@ -0,0 +1,45 @@
+=============================
+dbxincluder API documentation
+=============================
+
+Internals
+=========
+
+dbxincluder is divided into two parts.
+
+The xinclude submodule handles the pure XInclude 1.1 specification.
+It was necessary to re-implement it in python as libxml2 does not handle all necessary XInclude 1.1 features itself.
```

Comment:
* two "necessary"s; maybe rewrite it like: "...as libxml2 doesn't support all XInclude 1.1 features [itself]."
* python -> Python